dspace-spring-rest/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -89,11 +89,15 @@ public void findAllTest() throws Exception {
         getClient(token).perform(get("/api/core/items"))
                    .andExpect(status().isOk())
                    .andExpect(jsonPath("$._embedded.items", Matchers.containsInAnyOrder(
```

Comment:
I'm not able to find any real changes other than code format in this file. Is any?

Reply:
One small change.  The operation value is now a string ("true"), not a boolean.  

The reformatting in this file was due to a checkstyle error (exceeded 120 character limit). In some cases the reformatting was probably not needed (caused by erroneous indentation, sorry!).